src/detections/fetch/repository/get.js

```diff
@@ -0,0 +1,8 @@
+import { from } from 'rxjs'
+import { map } from 'rxjs/operators'
+
+import { GET_DETECTIONS } from 'detections/fetch/repository/query'
+import { client } from 'startup/client'
+
+export const get = payload => from(client().query({ query: GET_DETECTIONS, variables: { parameter: payload } }))
```

Comment:
Let's extract this query building business into a separate function.

Reply:
Done